CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## Unreleased
 
+### Changed
+
+- ([Change Log](https://github.com/lingua-pupuli/puppet-editor-services/releases/tag/0.13.0)) Updated Puppet Editor Services to version 0.13.0
```

Comment:
would rather (maint) than (ChangeLog). Either that or (Puppet Editor Services)

Reply:
Fixed